hack/arktos-up-scale-out-poc.sh

```diff
@@ -0,0 +1,609 @@
+#!/usr/bin/env bash
```

Comment:
do we have a similar script already to reuse ?

Reply:
This script is a copy over from Peng's changes. He would work on merging it into master and maximize code reuse. We will pick up his changes from master once it is merged.